src/main/java/ch/admin/bag/covidcertificate/service/KpiDataService.java

```diff
@@ -187,6 +192,24 @@ public void logExceptionalCertificateGenerationKpi(
                                     usedKeyIdentifier);
     }
 
+    @Transactional
+    public void logCertificateConversionKpi(
+            VaccinationCertificateConversionRequestDto conversionDto,
+            String newUvci,
+            String usedKeyIdentifier) {
+
+        logCertificateConversionKpi(
+                conversionDto.getDecodedCert()
+                             .getVaccinationInfo()
+                             .get(0)
+                             .getCountryOfVaccination(), // existing country column
+                // existing column system source
+                newUvci, // existing UVCI column
+                conversionDto.getDecodedCert().getVaccinationInfo().get(0).getIdentifier(), // new origin_uvci column
+                usedKeyIdentifier,
+                conversionDto.getConversionReason()); // conversion reason gets inserted to details column
```

Comment:
Der Kommentar ist nicht korrekt, oder? Der Conversion Reason ist soweit ich das sehe in einer separaten Spalte gespeichert und *nicht* in den Details.

Reply:
Ist angepasst.